pyccel/stdlib/math/pyc_math_c.c

```diff
@@ -3,7 +3,7 @@
 /* or go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details. */
 /* --------------------------------------------------------------------------------------- */
 
-#include "pyc_math.h"
+#include "pyc_math_c.h"
```

Comment:
```c
#ifndef         PYC_MATH_H
#define         PYC_MATH_H
```
to be changed to 
```c
#ifndef         PYC_MATH_C_H
#define         PYC_MATH_C_H
```
to avoid infinite include-loop.

Reply:
The infinite loop is already avoided (the name of the macro can be whatever), but you are right it is much better if this matches the file name. Fixed in 0762ad1c